xml/grammar.js

```diff
@@ -117,13 +122,20 @@ module.exports = grammar(DTD, {
 
     STag: $ => seq(
       '<',
-      $.Name,
+      alias($._start_tag_name, $.Name),
       c.rseq($._S, $.Attribute),
       O($._S),
       '>'
     ),
 
-    ETag: $ => seq('</', $.Name, O($._S), '>'),
+    ETag: $ => seq('</', alias($._end_tag_name, $.Name), O($._S), '>'),
+
+    _ErroneousETag: $ => seq(
```

Comment:
Is it possible to replace this with an `ERROR` node?

Reply:
not exactly, you could alias it though

Reply:
Would tree-sitter count it as an error node?

Reply:
nah it wouldn't, if you mean the error symbol under the hood